core/trino-main/src/main/java/io/trino/testing/TestingAccessControlManager.java

```diff
@@ -616,7 +616,7 @@ public void checkCanGrantExecuteFunctionPrivilege(SecurityContext context, Funct
             denyGrantExecuteFunctionPrivilege(functionName.toString(), context.getIdentity(), grantee);
```

Comment:
Can you please provide a test for that?

Reply:
`TestTestingAccessControlManager`? :)

Reply:
But the call to `shouldDenyPrivilege` should probably include `functionKind` somehow.

Reply:
I was thinking about `io.trino.security.TestAccessControl`

Reply:
Ok, IMO test in `io.trino.security.TestAccessControl` is not a good solution because those tests checks if correct `shouldDenyPrivilege` from `TestingAccessControlManager` is called, and check nothing related to underneath `AccessControlManager`.
Actually test like that already exists and before and after my fix it's still green.

@ksobolew has a nice idea to use `io.trino.spi.testing.InterfaceTestUtils#assertProperForwardingMethodsAreCalled(java.lang.Class<I>, java.util.function.Function<I,C>)` but instead of delegating `TestingAccessControlManager` extends from `AccessControlManager` so I cannot use that nice helper.

@kokosing any ideas?

Reply:
I have no more ideas.. :(